docs/docs/manual/columnediting.md

```diff
@@ -0,0 +1,128 @@
+
+Fetching allows you to set a “throttle delay” which determines the amount of time between requests. The default is 5 seconds per row in your dataset (5000 milliseconds), so you can estimate how long it will take to work through the entire column using that number. We recommend leaving this value at 5000 or greater. 
+
+![A screenshot of the settings window for fetching URLs.](/img/fetchingURLs.png)
+
+Note the following:
+
+* Many systems prevent you from making too many requests per second. To avoid this problem, set the throttle delay, which tells OpenRefine to wait the specified number of milliseconds between URL requests.
+* Before pressing OK, copy/paste a URL or two from the right column in the dialog and test them in another browser tab to make sure they work.
+* In some situations you may need to set[ HTTP request headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers). To set these, click the small “Show” button next to "HTTP headers to be used when fetching URLs" in the settings window. You can set the following request headers:
+  * [User-Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent)
+  * [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept)
+  * [Authorization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization)
+
+### Common errors
+
+When OpenRefine attempts to fetch information from a web page or service, it can fail in a variety of ways. The following information is meant to help troubleshoot and fix problems encountered when using this function.
+
+First, make sure that your fetching operation is storing errors (check “store error”). The run the fetch and look at the error messages. 
+
+**"Received fatal alert: handshake_failure" **can occur when you are trying to retrieve information over HTTPS but the remote site is using an encryption not supported by the Java virtual machine being used by OpenRefine.
+
+You can check which encryption methods are supported by your OpenRefine/Java installation by using a service such as **How's my SSL**. Add the URL `https://www.howsmyssl.com/a/check` to an OpenRefine cell and run "Add column by fetching URLs" on it, which will provide a description of the SSL client being used. 
+
+You can try installing additional encryption supports by installing the [Java Cryptography Extension](https://www.oracle.com/java/technologies/javase-jce8-downloads.html). Note on OpenRefine for Mac, these updated cipher suites need to be dropped into the Java install within the OpenRefine application: something like `/Applications/OpenRefine.app/Contents/PlugIns/jdk1.8.0_60.jdk/Contents/Home/jre/lib/security`.
```

Comment:
In addition to Mac, also Windows users who are using the OpenRefine kit with the bundled JRE.

I'm not sure this merits such a prominent mention though. Historically when this happens there is a cluster of problems for a lot of users, but it's a relatively infrequent occurrence.

Reply:
Yes, I wondered the same - not having anecdotal experience with any of these errors, I'm glad someone piped up. 

Having reordered with the 4xx/5xx errors first, can any of these be eliminated from the list as no longer being "common"?

Reply:
FYI to all, these are the current error messages listed - this is your chance to let me know if anything needs to be removed or added: 

"HTTP error 403 : Forbidden" 
"HTTP error 404 : Not Found" 
"HTTP error 500 : Internal Server Error"
“error: javax.net.ssl.SSLHandshakeException: Received fatal alert: handshake_failure”
"sun.security.validator.ValidatorException: PKIX path building failed"

Reply:
You mean just for the **Fetch URLs** operation however, right?  because there's also about 250+ error message kinds for Transforming data that could get potentially stored in a cell depending on if you checked the radio button to store the error. ;-)